python/lsst/ts/observatory/control/base_calsys.py

```diff
@@ -188,6 +205,25 @@ def load_calibration_config_file(self, filename: str | None = None) -> None:
         with open(data_path, "r") as f:
             self.calibration_config = yaml.safe_load(f)
 
+        config_schema = None
+        try:
```

Comment:
we are providing the schema ourselves so there's no reason to do this try/except here. If we can't open the file it should error out.